hudi-common/src/main/java/org/apache/hudi/avro/HoodieAvroUtils.java

```diff
@@ -202,6 +202,21 @@ public static byte[] avroToJson(GenericRecord record, boolean pretty) throws IOE
     return out.toByteArray();
   }
 
+  /**
+   * Convert a given avro record to json and return the string
+   *
+   * @param record The GenericRecord to convert
+   * @param pretty Whether to pretty-print the json output
+   */
+  public static String avroToJsonString(GenericRecord record, boolean pretty) throws IOException {
```

Comment:
Can we reuse existing `public static byte[] avroToJson(GenericRecord record, boolean pretty)`?